mappings/net/minecraft/client/item/TooltipContext.mapping

```diff
@@ -1,4 +1,14 @@
 CLASS net/minecraft/unmapped/C_taebrtdw net/minecraft/client/item/TooltipContext
+	FIELD f_yceowjjc NON_ADVANCED_NON_CREATIVE Lnet/minecraft/unmapped/C_taebrtdw$C_vsiqutyg;
+	FIELD f_zuchejxz ADVANCED_NON_CREATIVE Lnet/minecraft/unmapped/C_taebrtdw$C_vsiqutyg;
 	METHOD m_ecuysqaa isAdvanced ()Z
+	METHOD m_wmqdlnmf isCreative ()Z
```

Comment:
```suggestion
	METHOD m_wmqdlnmf isInCreativeTab ()Z
```
Not 100% sure but i think this is for when and only when the stack is in the tab, so it should be a bit clearer

Reply:
that seems right based on a quick investigation, will commit the suggestion!